ansible_collections/arista/avd/roles/eos_designs/templates/network_services/default-vrf-route-map.j2

```diff
@@ -0,0 +1,17 @@
+{# router_bgp.prefix_lists #}
```

Comment:
not router_bgp.prefix_lists. Just prefix_lists. Same below for route-maps.

Reply:
My bad. Will change the comment.